src/drivers/Qt/ConsoleVideoConf.cpp

```diff
@@ -132,7 +132,7 @@ ConsoleVideoConfDialog_t::ConsoleVideoConfDialog_t(QWidget *parent)
 
 	vbox1->addWidget( gl_LF_chkBox );
 
-#ifdef WIN32
+#if defined(WIN32) && (QT_VERSION_MAJOR < 6)
 	// 1px full screen border - hack fix for QOpenGLWidget fullscreen issues
 	winFullScreenBorderCbx  = new QCheckBox( tr("Fullscreen Border (1px)") );
```

Comment:
If the checkbox is to be removed for Qt 6 builds, then the winFullScreenBorderCbx variable should not exist at all. The header should be modified as well.

Reply:
Fixed this.